tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java

```diff
@@ -406,6 +410,7 @@ private static void testMouseAction(MouseAction mouseAction, MouseButton mouseBu
             switch (mouseAction) {
                 case PRESSED:
                     robot.mousePress(mouseButton);
+                    pressedButtons.add(mouseButton);
```

Comment:
As long as this is the only place you press without releasing a mouse button, then this looks good. I'll test it.

Reply:
Besides the exception tests (which should trigger exceptions and therefore the mouse button will never be pressed) I believe it is.